app/components/works/agreement_component.rb

```diff
@@ -7,10 +7,6 @@ def initialize(form:)
       @form = form
     end
 
-    def agree_to_terms?
-      work_form.agree_to_terms
-    end
-
     def work_form
       form.object
     end
```

Comment:
do we still use work_form?  I'm guessing it can be removed?

Reply:
the `work_form` appears to be used to fetch the `work`, which is then used to get to the depositor (some lines not visible in the diff)